components/org.wso2.carbon.identity.oauth.uma.common/src/main/java/org/wso2/carbon/identity/oauth/uma/common/HandleErrorResponseConstants.java

```diff
@@ -28,13 +28,17 @@
  */
 public final class HandleErrorResponseConstants {
 
-    public static final Map<String, String[]> RESPONSE_DATA_MAP = new HashMap<String, String[]>() {
-        {
-            put("60001", new String[]{"404", "not_found"});
-            put("60002", new String[]{"400", "invalid_request"});
-            put("60003", new String[]{"409", "conflict"});
-            put("60004", new String[]{"400", "invalid_resource_id"});
-            put("60005", new String[]{"400", "invalid_scope"});
-        }
-    };
+    private static final Map<String, String[]> RESPONSE_DATA_MAP = new HashMap<>();
+
+    public static Map<String, String[]> initializeResponseDataMap() {
```

Comment:
We can do the initialization inside a static block.

Something like,
static {
            RESPONSE_DATA_MAP.put("60001", new String[]{"404", "not_found"});
            RESPONSE_DATA_MAP.put("60002", new String[]{"400", "invalid_request"});
}

or another option is to use an ENUM to contain the Error details. You can find a sample from https://github.com/wso2/identity-api-server/blob/1b0afe2fe1308e71b678edf66864c947f0c45aa9/components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.common/src/main/java/org/wso2/carbon/identity/api/server/idp/common/Constants.java#L53

Reply:
Added a static initialization block.